test/test.go

```diff
@@ -0,0 +1,213 @@
+		"STD": Locale{"Dobra", 0, empty, empty, "", empty, empty, "DB", true},
+		"SVC": Locale{"El Salvador Colon", 2, ",", ".", "", "20A1", "&#x20A1;", "¢", true},
+		"SYP": Locale{"Syrian Pound", 0, empty, empty, "", "00A3", "&#x00A3;", "£", true},
+		"SZL": Locale{"Lilangeni", 2, "", ".", "", empty, empty, "E", true},
+		"THB": Locale{"Baht", 2, ",", ".", " ", "0E3F", "&#x0E3F;", "Bt", false},
+		"TJR": Locale{"Tajik Ruble", 0, empty, empty, "", empty, empty, "TJR ", true},
+		"TJS": Locale{"Somoni", 0, empty, empty, empty, empty, empty, "TJS ", true},
+		"TMM": Locale{"Manat", 0, empty, empty, "", empty, empty, "T", true},
+		"TND": Locale{"Tunisian Dinar", 3, empty, empty, "", empty, empty, "TND ", true},
+		"TOP": Locale{"Pa'anga", 2, ",", ".", " ", empty, empty, "$", true},
+		"TPE": Locale{"Timor Escudo", 0, empty, empty, empty, empty, empty, "TPE ", true},
+		"TRY": Locale{"Turkish Lira", 0, ",", "", "", "20A4", "&#x20A4;", "TL", false},
+		"TTD": Locale{"Trinidad and Tobago Dollar", 0, empty, empty, "", "0024", "&#x0024;", "TT$", true},
+		"TWD": Locale{"New Taiwan Dollar", 0, empty, empty, "", "0024", "&#x0024;", "NT$", true},
+		"TZS": Locale{"Tanzanian Shilling", 2, ",", ".", " ", empty, empty, "TZs", false},
+		"UAH": Locale{"Hryvnia", 2, " ", ",", "", "20B4", "&#x20B4", "UAH ", false},
+		"UGX": Locale{"Uganda Shilling", 0, empty, empty, "", empty, empty, "UGX", true},
+		"USD": Locale{"US Dollar", 2, ",", ".", "", "0024", "&#x0024;", "$", true},
+		"UYU": Locale{"Peso Uruguayo", 2, ".", ",", "", "20B1", "&#x20B1;", "$", true},
+		"UZS": Locale{"Uzbekistan Sum", 0, empty, empty, "", empty, empty, "лв", true},
+		"VEF": Locale{"Bolivar", 2, ".", ",", " ", empty, empty, "Bs.", true},
+		"VND": Locale{"Dong", 2, ".", ",", " ", "20AB", "&#x20AB;", "₫", true},
+		"VUV": Locale{"Vatu", 0, ",", "", "", empty, empty, "VT", false},
+		"WST": Locale{"Tala", 0, empty, empty, "", empty, empty, "WST ", true},
+		"XAF": Locale{"CFA Franc BEAC", 0, empty, empty, "", empty, empty, "$", true},
+		"XCD": Locale{"East Caribbean Dollar", 2, ",", ".", "", "0024", "&#x0024;", "$", true},
+		"XOF": Locale{"CFA Franc BCEAO", 0, empty, empty, empty, empty, empty, "XOF ", true},
+		"XPF": Locale{"CFP Franc", 0, empty, empty, "", empty, empty, "XPF ", true},
+		"YER": Locale{"Yemeni Rial", 0, empty, empty, "", "FDFC", "&#xFDFC;", "﷼ ", true},
+		"YUN": Locale{"New Dinar", 0, empty, empty, "", empty, empty, "YUN ", true},
+		"ZAR": Locale{"Rand", 2, " ", ".", " ", "0052", "&#x0052;", "R", true},
+		"ZMK": Locale{"Kwacha", 0, empty, empty, "", empty, empty, "ZMK ", true},
+		"ZRN": Locale{"New Zaire", 0, empty, empty, empty, empty, empty, "ZRN ", true},
+		"ZWD": Locale{"Zimbabwe Dollar ", 2, " ", ".", "", "0024", "&#x0024;", "Z$", true},
+	}
+
+	lc := LocaleInfo["INR"]
+	ac := accounting.Accounting{Symbol: lc.ComSymbol, Precision: 2, Thousand: lc.ThouSep, Decimal: lc.DecSep}
+	fmt.Println(ac.FormatMoney(500000)) // "$500,000.00"
```

Comment:
I have some comments for this test 😄 

1. Could you change the code to use `AssertEqual` instead of just printing it like below line? 
https://github.com/leekchan/accounting/blob/17a4ce5f94d4d6b8115c2c6f39f7f2e8cb8f27f0/formatnumber_test.go#L12

2. And I think you can move this test into the formatnumber_test.go file because this code actually tests the formatnumber function. 

3. I think you don't need to copy & paste the Locale struct into the test file. I think testing`FormatNumber` directly with `"₹"` parameter is enough because it is `unit test` 😄